sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -349,7 +353,7 @@ public Mono<Response<Void>> deleteTableWithResponse() {
     }
 
     Mono<Response<Void>> deleteTableWithResponse(Context context) {
-        context = context == null ? Context.NONE : context;
+        context = setContext(context);
```

Comment:
Shouldn't we also attach the tracing context to this when sending requests to the service?

Reply:
Can you elaborate on tracing context? I'm not quite sure what you're referring to? @samvaity

Reply:
context.addData(AZ_TRACING_NAMESPACE_KEY, CONTAINER_REGISTRY_TRACING_NAMESPACE_VALUE)
https://github.com/Azure/azure-sdk-for-java/blob/54ff62528d282f885c2022b497bd6a385c0e219a/sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/UtilsImpl.java#L89